core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -585,6 +585,7 @@ public interface MetricsProducer {
   String METRICS_COMPACTOR_MAJC_STUCK = METRICS_COMPACTOR_PREFIX + "majc.stuck";
   String METRICS_COMPACTOR_ENTRIES_READ = METRICS_COMPACTOR_PREFIX + "entries.read";
   String METRICS_COMPACTOR_ENTRIES_WRITTEN = METRICS_COMPACTOR_PREFIX + "entries.written";
+  String METRICS_COMPACTOR_BUSY = METRICS_COMPACTOR_PREFIX + "busy";
```

Comment:
Needs an metric definition added in the above javadoc to describe the metric (type, value, etc).

Reply:
added in e8f8e12